data/graphics.js

```diff
@@ -13592,6 +13592,34 @@ var BattleMoveAnims = {
 			var attacker = args[0];
 			var defender;
 
+			battle.bgElem.animate({
+				position: 'absolute',
+				left: -5,
```

Comment:
The `bgElem` starts at `-90, -50`. Don't you notice it being positioned weirdly if you do this? Have you tested a gen 1 replay with Earthquake?